src/Components/Buttons/ReverseList.tsx

```diff
@@ -0,0 +1,20 @@
+import cn from 'classnames';
+import { ReverseListProps } from '../../types/buttons/ReverseListProps';
+import React from 'react';
+
+export const ReverseList: React.FC<ReverseListProps> = ({
+  isReversed,
+  setIsReversed,
+}) => {
+  return (
+    <button
+      type="button"
+      className={cn('button is-warning', {
+        'is-light': !isReversed,
+      })}
+      onClick={() => setIsReversed(!isReversed)}
+    >
+      Reverse
+    </button>
```

Comment:
According to the checklist item [UX] - don't forget to mark the active button to explain to the user what action is active. The button does change its class based on the `isReversed` state, which is good. However, it's not clear if the visual change is enough to indicate the active state to the user. Consider adding an aria-pressed attribute or changing the button text to indicate the active state more clearly.